plugins/3scale-backend/app-config.yaml

```diff
@@ -0,0 +1,11 @@
+catalog:
+  providers:
+    threeScaleApiEntity:
+      dev:
+        baseUrl: '${3SCALE_BASE_URL}' # https://<TENANT>-admin.3scale.net
+        accessToken: '${3SCALE_ACCESS_TOKEN}'
+        schedule: # optional; same options as in TaskScheduleDefinition
+          # supports cron, ISO duration, "human duration" as used in code
+          frequency: { minutes: 1 }
```

Comment:
Do we want this to be 1 minute?

Reply:
I mainly copied back what was in the wrappers, which is what was in the initial static build of the showcase

Reply:
But we could decide to change it later on of course.

Reply:
I just copied what was in the [readme](https://github.com/janus-idp/backstage-plugins/blob/176f965b2565bbf81c130d30b21096ef2e70c9ab/plugins/3scale-backend/README.md)

Reply:
This would be the default config which can be overridden of course.
We could change the value later on of we decide it should be changed.

Reply:
Created issue https://github.com/janus-idp/backstage-plugins/issues/914 to track this.